### PR TITLE
Add new parameters from RouterOS 7.16 release

### PR DESCRIPTION
**Pull request by derdeagle**
###### /interface/bridge

- `forward-reserved-addresses` with default value `False`
- `max-learned-entries` with default value `auto`

###### /ip/neighbor/discovery-settings

- `lldp-mac-phy-config` with default value `False` (from Router OS 7.15 release)
- `discover-interval` with default value `30s`
- `lldp-vlan-info` with default value `False`

###### /ip/settings

- `ipv4-multipath-hash-policy` with default value `l3`

###### /ipv6/settings

- `multipath-hash-policy` with default value `l3`

###### /ip/dhcp-server/option

- `comment`

###### /ip/dhcp-server/option/sets

- `comment`

###### /ip/dhcp-server/matcher

- `comment`

###### /tool/netwatch

- `acccept-icmp-time-exceeded` with default value `False`
- `ttl` with defaut value `255`

###### /ip/service

- `max-sessions` with default value `20`

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: changelogs/fragments/323-add-ros-7.16-parameters.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_info, api_modify - Add new parameters from the RouterOS 7.16 release (https://github.com/ansible-collections/community.routeros/pull/323).
```
Comment (felixfontein) [inline changelogs/fragments/323-add-ros-7.16-parameters.yml:2]:
```suggestion
  - api_info, api_modify - add new parameters from the RouterOS 7.16 release (https://github.com/ansible-collections/community.routeros/pull/323).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

**Comment by felixfontein** [comment]
@derdeagle thanks!

**Comment by derdeagle** [comment]
My pleasure ;)

**Pull request merged**